Makefile

```diff
@@ -104,11 +104,11 @@ help: ## Display this help.
 
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="{./api/v1,./cmd/manager,./cmd/metrics,./controllers}" output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="{./api/v1,./cmd/manager,./cmd/metrics,./controllers,./config/peerpods/podvm}" output:crd:artifacts:config=config/crd/bases
```

Comment:
For the records : as discussed in slack this would defeat https://github.com/openshift/sandboxed-containers-operator/pull/678/commits/bc47b2c5e13eba1d8c46a435d2e12a60eb7bdfa4#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R107 . The `+kubebuilder` annotation in `config/peerpods/podvm/extra_rbac.go` should be added somewhere else.